u-boot/pkg.yaml

```diff
@@ -26,10 +26,13 @@ steps:
     env:
       SUN50I_A64_ARM_TRUSTED_FIRMWARE: sun50i_a64_arm-trusted-firmware
       RK3328_ARM_TRUSTED_FIRMWARE: rk3328_arm-trusted-firmware
+      RK3399_ARM_TRUSTED_FIRMWARE: rk3399_arm-trusted-firmware
       LIBRETECH_ALL_H3_CC_H5_U_BOOT: libretech_all_h3_cc_h5_u-boot
       ROCK64_RK3328_U_BOOT: rock64_rk3328_u-boot
       RPI_ARM64_U_BOOT: rpi_4_u-boot
       BANANAPI_M64_U_BOOT: bananapi_m64_u-boot
+      ROCK_PI_4_RK3399_U_BOOT: rock-pi-4-rk3399_u-boot
+      M0_CROSS_COMPILE: /toolchain/bin/
```

Comment:
I don't see where this is used?

Reply:
well, it's used in u-boot makefiles but as we skip the build I guess I should drop it, thanks :+1: